README.md

```diff
@@ -602,6 +602,7 @@ Table of Contents
   * [cloudsploit.com](https://cloudsploit.com) — AWS security and configuration monitoring. Free: unlimited on-demand scans, unlimited users, unlimited stored accounts. Subscription: automated scanning, API access, etc.
   * [datadoghq.com](https://www.datadoghq.com/) — Free for up to 5 nodes
   * [deadmanssnitch.com](https://deadmanssnitch.com/) — Monitoring for cron jobs. 1 free snitch (monitor), more if you refer others to sign up
+  * [economize.cloud](https://economize.cloud) — Free for up to $5000 spends on Google Cloud Platform.
```

Comment:
Generally (and i know its not true for all) we put a bit of description of what the thing does in addition to whats free

Then I am not sure does this 5000 mean first 5000 seen over al time is free or that if your spend is less than 5k monthly it would be free forever? Only if its the 2nd would it be good for this list

Reply:
Hi @ripienaar :wave: 

The $5000 cost tracking is free monthly-wise. The same has been updated on the website as well. You can have a look now.

Reply:
Hi @ripienaar :wave: 

I have added an extended description. Can you please have a look now?